src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserStateObjectFactory.Windows.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace System.Data.SqlClient
 {
-    internal sealed class TdsParserStateObjectFactory
+    internal sealed partial class TdsParserStateObjectFactory
```

Comment:
@saurabh500 @danmosemsft ah we don't need it actually (especially since we decided to use .Unix.cs version on Windows). Sorry I was on vacation last two weeks. Let me remove this change.